torch/_torch_docs.py

```diff
@@ -2009,6 +2009,13 @@ def merge_dicts(*dicts):
 Returns a new tensor with the hyperbolic cosine  of the elements of
 :attr:`input`.
 
+.. note::
```

Comment:
Let's move these notes after the example since they're unlikely to be important to the user. In the future we may want to consider a special kind of note or section, like "Implementation Details."